packages/timeline-state-resolver/src/devices/telemetrics.ts

```diff
@@ -166,6 +172,32 @@ export class TelemetricsDevice extends DeviceWithState<TelemetricsState, DeviceO
 			this.updateStatus(StatusCode.GOOD)
 			this.resolveInitPromise(true)
 		})
+
+		this.socket.on('ready', () => {
+			this.startSessionKeeper()
+		})
+	}
+
+	private startSessionKeeper(): void {
+		if (!this.sessionKeeperTimer) {
+			return
+		}
```

Comment:
I haven't tested the code, but this look like it will never start?
Shouldn't the `!` be removed?